### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by dannytrigo** [miracle2k/onkyo-eiscp#156]
I am starting to use this project to build a better integration for Onkyo receivers in Home Assistant, but I see a pull request from 2021 with no activity. Is there a new fork which is more actively maintained, or should I stick to using my own fork?

**Comment by rlKoekie** [comment]
Hi @dannytrigo, I'm also dreaming of some improvements in the Onkyo department of Home Assistant. I would love to be able to control the multiroom audio / flareconnect setup from home assistant. 
Did you find a way forward already? If so, I would love to see my pull request included: https://github.com/miracle2k/onkyo-eiscp/pull/159 , this adds a function to easily set up the flareconnect zone, without having to send raw commands to the receiver.

**Comment by dannytrigo** [comment]
Hi @rlKoekie. So, I've forked this project to do some changes/bugfixes I needed (https://github.com/dannytrigo/onkyo-eiscp)
And I've been submitting many PRs to this project which was the beginnings of a new Onkyo integration for HA: (https://github.com/kongo09/hass-onkyo-ng).

I've got it into a decent state at least for my own use - its responsive to state changes, supports multi-zones, etc. but there's still plenty of room for improvement. I can definitely include your PR into my fork of this project, so it gets into that integration.

I only own one Onkyo, so won't be able to test, but that sounds like a great candidate to be a HA service. You could call the service on one receiver entity ID and pass the entity IDs of the other receivers to have them create a flareconnect group.

**Comment by rlKoekie** [comment]
Hi @dannytrigo , I'm installing HACS now to have a look at the hass-onkyo-ng integration. I see you already created the pull request in your onkyo-eiscp fork, so I'll keep an eye on both projects. (I'm guessing you first need to update the onkyo-eiscp fork, then publish a new version to pypi, before we can start working on the multiroom audio part in the hass integration).
I'm more than happy to do the testing of multiroom audio for you, I've got an TX-NR676E and a NCP-302, but I'll move that conversation to https://github.com/kongo09/hass-onkyo-ng :-)

**Comment by dannytrigo** [comment]
Yep, I'll get it integrated into my fork, then we can include it in hass-onkyo-ng and get to work :)

**Comment by Nebula83** [comment]
What a coincidence: just this week I started to work on improving the Onkyo integration as well. I need listening modes and audissey sound something. I currently have a config a d options flow working and an autodiscovery set up. Can I chip in?

**Comment by dannytrigo** [comment]
I’m not the repository owner, but although the owner is not actively maintaining it, he has been happy to integrate my PRs. So I’d definitely say combine efforts and open PRs on the above repo :)

And it already gets listening modes :)

**Comment by Nebula83** [comment]
Gave `hass-onkyo-ng` a try, but it didn't for my model. I decided to update the official integration to a more current state. Maybe that's a nice starting point for the other integration to roll back into as well.

**Comment by mitchcapper** [comment]
I recommend my PR #160
